introduction.md

```diff
@@ -0,0 +1,9 @@
+Hello my name is Guillaume and i'm a young french developper, and i'm trying
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?